styles.css

```
@@ -1 +1,82 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #FEFAE0;
+}
+
+.container {
+    margin: 20px;
+    background-color: #FEFAE0;
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 50px auto 50px 50px 50px 50px 50px;
+    row-gap: 16px;
+    column-gap: 16px;
+    grid-template-areas:
+        "a"
+        "b"
+        "c"
+        "d"
+        "e"
+        "."
+        "f";
+}
+
+.item {
+    background-color: #283618;
+    color: #FEFAE0;
+    border: 5px;
```

@abouman76 - border krijgt een hsl color-eigenschap:
```suggestion
    border: 2px solid hsl(355, 52%, 38%);
```